app-network/mihomo-party/autobuild/postinst

```diff
@@ -0,0 +1,7 @@
+if hash update-mime-database 2>/dev/null; then
+    update-mime-database /usr/share/mime || true
+fi
+
+if hash update-desktop-database 2>/dev/null; then
+    update-desktop-database /usr/share/applications || true
+fi
```

Comment:
None of this is necessary (did you copy all these from Arch?). We have triggers to handle this.